dwpost/dwpost.php

```diff
@@ -47,7 +50,7 @@ function dwpost_jot_nets(&$a,&$b) {
 }
 
 
-function dwpost_settings(&$a,&$s) {
+function dwpost_settings(App $a, $s) {
```

Comment:
The `&` operator of the `$s` parameter enables this function to change the hook data. If you remove it this function has no effect whatsoever.